src/lib/markbind/src/utils/urls.js

```diff
@@ -44,12 +44,42 @@ function calculateNewBaseUrls(filePath, root, lookUp) {
   return calculate(filePath, undefined);
 }
 
+/**
+ * Returns the absolute path of of the immediate parent site of the specified file path.
+ */
+function calculateImmediateParentSitePath(filePath, root, lookUp) {
```

Comment:
Simpler version of `calculateNewBaseUrls` that just returns the immediate parent site path, which is what's needed for the majority of use cases